.mergify.yml

```diff
@@ -11,6 +11,7 @@ queue_rules:
           - -files~=^[^/]*\.yaml$
           - -files~=^data/.*\.yaml$
         - and:
+          - check-success=Haskell-CI - Linux - ghc-9.2.4
```

Comment:
This was missing before, which I assume was just an oversight?  Or was there a good reason not to put `ghc-9.2` in the `queue_rules`?

Reply:
I think it was failing more often when the rule was written. But it doesn’t make sense to check it before and not after so it’s probably a mistake.

Anyway 9.2 has been very stable for a long time, so there is no reason to not check it. 😅